api_tests/preprints/views/test_preprint_citations.py

```diff
@@ -49,9 +50,23 @@ def test_citation_url_is_preprint_url_not_project(self):
 		assert_equal(res.json['data']['links']['self'], display_absolute_url(self.published_preprint))
 
 
-class TestPreprintCitationsStyle(PreprintCitationsMixin, ApiTestCase):
+class TestPreprintCitationContent(PreprintCitationsMixin, ApiTestCase):
 
 	def setUp(self):
-		super(TestPreprintCitationsStyle, self).setUp()
+		super(TestPreprintCitationContent, self).setUp()
 		self.published_preprint_url = '/{}preprints/{}/citation/apa/'.format(API_BASE, self.published_preprint._id)
 		self.unpublished_preprint_url = '/{}preprints/{}/citation/apa/'.format(API_BASE, self.unpublished_preprint._id)
+
+	def test_citation_contains_correct_date(self):
+		res = self.app.get(self.published_preprint_url)
+		assert_equal(res.status_code, 200)
+		expected_date = self.published_preprint.date_modified.strftime('%Y, %B %-d')
+		assert_true(expected_date in res.json['data']['attributes']['citation'])
+
+		self.published_preprint.original_publication_date = datetime(2017, 12, 24)
```

Comment:
If this is a datetime rather than just a date, be sure to use a timezone for this, just to ensure that there won't be any future test problems. It's not risky in this case, but just in case.  If it definitely doesn't matter because we are naively just chopping off the time and tz info or something, then feel free to ignore.

Reply:
No time info is returned for these citations. They only return date info.